apps/rush-lib/src/cli/logic/InstallManager.ts

```diff
@@ -223,7 +223,9 @@ export default class InstallManager {
    */
   public createTempModulesAndCheckShrinkwrap(
     shrinkwrapFile: BaseShrinkwrapFile | undefined,
-    forceCreate: boolean): boolean {
+    forceCreate: boolean,
+    authTokens: string[] = []
```

Comment:
> = [] [](start = 24, length = 5)

I would avoid making this be optional

Reply:
+1

---
In reply to: [177545538](https://github.com/Microsoft/web-build-tools/pull/585#discussion_r177545538) [](ancestors = 177545538)